node/src/config/arguments.js

```diff
@@ -49,6 +50,14 @@ const argumentsMap = {
     prompt: 'Project ID',
     group: ['deploy', 'destroy', 'approve', 'cancel']
   },
+  [WORKSPACE_NAME]: {
+    name: WORKSPACE_NAME,
+    alias: 'e',
```

Comment:
we already have `e` alias.
We don't have to have an alias for every option.

Reply:
https://github.com/env0/env0-client-integrations/pull/53/commits/1294a05abe9dbf038de5a46d6a8b0cc7ef8aff6f